lib/proxy_manager.js

```diff
@@ -160,20 +160,34 @@ function loadConfigFiles(cb) {
     stop_me[proxy_config_file] = proxy;
   });
 
+  // TODO (@theopak): Consider rewriting this to make it recursive.
   fs.readdir(CONFIG_DIR, function(err, files) {
     /* istanbul ignore if */
     if (err) return cb(err);
 
+    // Require a '.json' or '.xml' file ending and reject dotfiles
```

Comment:
Oh, I don't remember what I was thinking. I'll fix this line. I'll also remove the TODO comment since it's done.